File: Prime Number.cs
using System;

class program{
	public static void Main(){
		int CurrentVal = 2;
		while (true){
			bool Prime = true;
			for(int i=2;i<CurrentVal;i++){
				if (CurrentVal%i == 0){
					Prime = false;
				}
			}
			if (Prime == true){
				Console.WriteLine($"{CurrentVal} Is Prime");
			}
			CurrentVal += 1;
		}
	}
}

File: primenumber.cpp
#include <iostream>

int main(){
	int currentval = 2;
	while (true){
		bool prime = true;
		for (int i=2;i<currentval;i++){
			if (currentval % i == 0){
				prime = false;
			}
		}
		if (prime == true){
			std::cout << currentval << " is prime" << std::endl;
		}
		currentval += 1;
	}		
}
